frontend/components/App/FishbowlPreJoin/form.tsx

```diff
@@ -59,14 +60,14 @@ const FormValidation = withFormik<FormProps, FormValues>({
   }
 })(Form);
 
-const Nickname = () => {
+const Nickname: React.FC = () => {
```

Comment:
And this one could be GuestForm

Reply:
It is only one field, should we use NicknameField instead of `Form` word?

Reply:
It is not only a field, it contains the whole logic about the form.
What is strange IMO is that on the form-auth there is only one component, and here we have one component for the "form" (without any logic) and another component.

Having 2 components on the same file seems not a great idea. (why can't we join them?)